framework/include/utils/ADRankFourTensorForward.h

```diff
@@ -0,0 +1,18 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ADRealForward.h"
+
+template <typename>
+class RankFourTensorTempl;
+
+typedef RankFourTensorTempl<Real> RankFourTensor;
```

Comment:
Our goal is to reduce forward declaration duplication, so I'd suggest replacing these declarations with `#include "RankFourTensorForward.h"`. Equivalently for `ADRankThreeTensorForward.h` and `ADRankTwoTensorForward.h`

Reply:
Done.